app/components/Modal/WithdrawModalNew.jsx

```diff
@@ -901,7 +901,7 @@ class WithdrawModalNew extends React.Component {
               !address ||
               !canCoverWithdrawal ||
               addressError ||
-              quantity < minWithdraw;
+              quantity <= minWithdraw;
```

Comment:
What was the reason here? 

if quantity == minWithdraw, the withdrawl should be possible, non?

Reply:
This is another fix *by the way* as the API is feeding *minWithdraw* and it should be "or equal" as well.

Reply:
With your fix, if quantitiy == minWithdraw, then withdrawal is NOT possible. Is that the intention?

Reply:
My intention is "equal" or "above", perhaps "<=" is wrong? not sure but I remember using "<=" for that intension.

Reply:
Thanks for the catch; just noticed that it should be "=>".

Reply:
the change was reverted back.